core/src/main/java/org/elasticsearch/action/admin/cluster/health/TransportClusterHealthAction.java

```diff
@@ -71,7 +72,14 @@ protected ClusterHealthResponse newResponse() {
     }
 
     @Override
-    protected void masterOperation(final ClusterHealthRequest request, final ClusterState unusedState, final ActionListener<ClusterHealthResponse> listener) {
+    protected final void masterOperation(ClusterHealthRequest request, ClusterState state, ActionListener<ClusterHealthResponse> listener) throws Exception {
+        logger.warn("attempt to execute a cluster health operation without a task");
+        throw new UnsupportedOperationException("task parameter is required for this operation");
+    }
+
+    @Override
+    protected void masterOperation(final Task task, final ClusterHealthRequest request, final ClusterState unusedState, final
+    ActionListener<ClusterHealthResponse> listener) {
```

Comment:
Somehow IntelliJ code formatter thinks that this is a perfectly fine way to split this line in two :)